ffmeta/blueprints/web.py

```diff
@@ -135,7 +135,11 @@ def var_page(varname=None):
     neighbors = session.query(Variable).filter(Variable.group_id == this_variable.group_id).filter(Variable.name != this_variable.name).all()
     responses = session.query(Response).filter(Response.name == varname).group_by(Response.label).all()
     if responses:
-        responses = sorted(responses, key=lambda x: x.value, reverse=True)
+        """Ordering the responses object to display non-negative responses (non-missing) first, ordered by frequency
```

Comment:
Use formatting like this to help the reader (don't worry about pep8 complaining about exceeding 80 chars, 120 is far more reasonable):

```
        responses = \
            sorted(filter(lambda x: not x.value.startswith('-'), responses), key=lambda x: x.freq, reverse=True) + \
            sorted(filter(lambda x: x.value.startswith('-'), responses), key=lambda x: x.freq, reverse=True)
```